src/managers/SessionManager.ts

```diff
@@ -1,13 +1,17 @@
 import { MasterTick } from "core/masterTick";
-import { UUID } from "definitions/utils";
+import { int, UUID } from "definitions/utils";
 import { EventEmitter } from "utilities/EventEmitter";
 import { log } from "utilities/logger";
 
 class SessionManager {
     public readonly onSync: EventEmitter;
 
+    private information: Array<string|int> = [];
```

Comment:
I assume this change is not necessary, since now we have private information: SessionInformation = {
sessionId: UUID = "",
... = "",
... = -1, 
...
}
meaning this.information upon initialization contains default values of type SessionInformation.